docs/gaia/gaia.rst

```diff
@@ -76,6 +76,9 @@ Examples
 
 This query searches for all the objects contained in an arbitrary rectangular projection of the sky.
 
+WARNING: This method implements the ADQL BOX function that is deprecated in the latest version of the standard
+(ADQL 2.1,  see: https://ivoa.net/documents/ADQL/20231107/PR-ADQL-2.1-20231107.html#tth_sEc4.2.9).
+
 It is possible to choose which data release to query, by default the Gaia DR3 catalogue is used. For example::
 
   >>> from astroquery.gaia import Gaia
```

Comment:
This now throws a doctest failure, so better make it remote-data, too


```suggestion
.. doctest-remote-data::

  >>> from astroquery.gaia import Gaia
```

Reply:
Fine by me.